core/src/main/java/org/lflang/generator/c/CPreambleGenerator.java

```diff
@@ -78,15 +78,10 @@ public static String generateDefineDirectives(
     //         targetConfig.clockSyncOptions
     //     ));
     // }
-    if (targetConfig.threading) {
-      targetConfig.compileDefinitions.put("LF_THREADED", "1");
-    } else {
+    if (targetConfig.singleThreaded) {
       targetConfig.compileDefinitions.put("LF_UNTHREADED", "1");
```

Comment:
We should change this macro from `LF_UNTHREADED` to `LF_SINGLE_THREADED` in `reactor-c`.

Reply:
True, I'll change all occurrences of "unthreaded" to "single threaded".